configure.ac

```diff
@@ -17,7 +17,7 @@ AC_CHECK_SIZEOF(int)
 AS_IF([test "$ac_cv_sizeof_int" -lt 4],
     AC_MSG_ERROR([Integer type 'int' must be at least 4 bytes in size.]))
 
-# FLAC support checks
+# FLAC decoding support checks
```

Comment:
This should still just say FLAC support checks. The same library provides both encoding and decoding, even though we only use it for decoding.